composer.json

```diff
@@ -17,14 +17,12 @@
     "require": {
         "php":                  ">=5.3.6",
         "behat/mink":           "^1.7.1@dev",
-        "symfony/browser-kit":  "~2.3|~3.0|~4.0",
-        "symfony/dom-crawler":  "~2.3|~3.0|~4.0"
+        "symfony/browser-kit":  "^2.3|^3.4|~4.0|~5.0"
     },
 
     "require-dev": {
         "mink/driver-testsuite": "dev-master",
-        "symfony/debug": "^2.7|^3.0|^4.0",
```

Comment:
Please be careful when resolving conflicts. You basically reverted most of my recent improvements to the CI setup.